packages/ffe-file-upload-react/src/FileItem.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import { number, func, shape, object, string as stringType } from 'prop-types';
+
+const FileItem = props => (
+    <li key={props.index}>
```

Comment:
I don't see a purpose of providing a key here.
If we are mapping an array to several `FileItem`s somewhere else we would rather give a key to each `FileItem`, as that's where React actually needs it.